docs/content/3.components/form.md

```diff
@@ -93,14 +93,19 @@ props:
 
 ### Input Events
 
-The Form component automatically triggers validation when an input emits an `input`, `change`, or `blur` event.
+The Form component automatically triggers validation when an input emits an `input`, `change`, `blur` or `submit` event.
```

Comment:
The submit event is not related to inputs, so this change is a bit confusing especially since the `validate-on` prop no longer support `submit`